charts/osm/values.yaml

```diff
@@ -167,6 +167,11 @@ osm:
     # The specified tolerations allow pods to schedule onto nodes with matching taints.
     tolerations: []
 
+  # -- Experimental values. Behavior is not supported.
+  experimental:
+    # -- Enable the MeshRootCertificate to configure the OSM certificate provider.
+    enableMeshRootCertificate: false
```

Comment:
That makes sense to me. Should the flag be added to the MeshConfig?

Reply:
I originally hesitated to add it as a part of the featureFlag spec since I thought all values in that spec supported dynamic configuration. I've added `enableMeshRootCertificate` to the featureFlag spec and did not include it in the MeshConfig